tests/app/views/test_brief_response.py

```diff
@@ -324,6 +325,110 @@ def test_day_rate_should_be_less_than_service_max_price(self, live_dos_framework
         assert data == {'error': {'dayRate': 'max_less_than_min'}}
 
 
+class TestSubmitBriefResponse(BaseBriefResponseTest):
+
+    def setup_existing_brief_response(self, brief_response_data):
```

Comment:
I think our use of the convention is pretty patchy in our code, and my grasp of the convention is patchy at best, but should these internal helper methods be `_setup_existing_brief_response` and `_submit_brief_response`?